test/gtest/uct/ib/test_ud.cc

```diff
@@ -807,7 +818,7 @@ UCS_TEST_P(test_ud, ep_destroy_creq) {
     m_e1->connect_to_iface(0, *m_e2);
     short_progress_loop(TEST_UD_PROGRESS_TIMEOUT);
 
-    uct_ep_destroy(m_e1->ep(0));
+    m_e1->destroy_ep(0);
```

Comment:
maybe use it in other places as well instead of `uct_ep_destroy`

Reply:
then we could do `validate_flush()` that touches EP object